### DISCUSSION TITLE
/opt/netbox/venv/bin/activate is missing

### DISCUSSION THREAD

**Discussion by simo735** [netbox-community/netbox#7352] [Q&A] [answered]
Hello,

To create a Super User, we need first to enter the Python virtual environment created by the upgrade script:
`source /opt/netbox/venv/bin/activate`

but i notice that the folder venv is missing :

![image](https://user-images.githubusercontent.com/2478297/134514952-96df2755-c9b3-4cfe-aeac-493793c24b0f.png)

I use this documentation: https://netbox.readthedocs.io/en/stable/installation/3-netbox/

Any idea please ?   

Thank you in advance.

**Comment by candlerb** [comment, marked as answer]
That directory is created when you run the `./upgrade.sh` script, which is the previous step [here](https://netbox.readthedocs.io/en/stable/installation/3-netbox/#run-the-upgrade-script).

**Reply by simo735** [reply]
Thank you @candlerb
